blamo/lib/Export/CSVExporter.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:flutter/material.dart';
+import 'package:blamo/ObjectHandler.dart';
+import 'package:blamo/main.dart';
+import 'dart:io';
+import 'dart:async';
+import 'package:path_provider/path_provider.dart';
+import 'package:permission_handler/permission_handler.dart';
+
+class CSVExporter {
+  ObjectHandler objectHandler = new ObjectHandler();
+  StateData stateData;
+
+  LogInfo logInfo;
+  List<Unit> units;
+  List<Test> tests;
```

Comment:
this data
![image](https://user-images.githubusercontent.com/43728787/76691646-339f3780-660a-11ea-813e-6fe07966ea3e.png)

outputs this
![image](https://user-images.githubusercontent.com/43728787/76691647-3e59cc80-660a-11ea-8cf9-3a2a1c61dd25.png)

Reply:
![image](https://user-images.githubusercontent.com/43728787/76691690-baecab00-660a-11ea-9816-238ac352a111.png)

Reply:
![image](https://user-images.githubusercontent.com/43728787/76691812-3733be00-660c-11ea-85d3-d80ab9a062ad.png)

Reply:
This problem was caused by new line characters, should be parsed and replaced with blank spaces for now. We will need to ask the client if they prefer them there or not.